sig-security-assessments/Documentation/recipe-book.md

```diff
@@ -0,0 +1,52 @@
+project and is an expert in it. If you are requesting a Security Self-assessment you can reach out to the team in
+[#sig-security](https://kubernetes.slack.com/messages/sig-security) to see if there is someone interested in helping to
+champion the assessment.
+1.	Make sure that you and your counterpart each have a back-up person assigned who attends all the meetings and is 
+involved in the assessment. Two heads are better than one, and redundancy means that if someone can’t make a meeting 
+for whatever reason, that you can still make progress.
+1.	To get further participation, advertise in other SIG meetings. Aim for a minimum of 4 people (a project expert, a 
+security expert, and a backup for each), and a maximum of 8. This ensures that there is not overwhelming burden on any
+one individual but allows for a smooth decision-making process.
+1.	Create a Google doc for meeting notes for future and past meetings for tracking. Share these notes in 
+[#sig-security](https://kubernetes.slack.com/messages/sig-security), 
+[#tag-security](https://cloud-native.slack.com/archives/CDJ7MLT8S), and the appropriate Kubernetes Slack channel for 
+the project you are assessing.
+## Meetings
+- Once you have your collateral and your team together, set up a kickoff meeting with everyone. It is likely that not 
+everyone will show up. That’s ok!
+- Keep meeting notes using the 
+[Kubernetes community meeeting notes template](https://github.com/kubernetes/community/blob/master/events/community-meeting.md).
+- Create a recurring meeting series of about ~6 meetings (this is flexible and can be extended if necessary)
+- These sessions can be ad hoc and should be set for whatever timing works for the group.
+## The Assessment
+### Step One
+Draw a [data flow diagram](https://www.lucidchart.com/pages/data-flow-diagram) for the main workflows in the project. 
+This step will show you the areas you can potentially assess, and allow the security expert to quickly gain context 
+around how the project is structured - and where the biggest security concerns may be. 
+[Excalidraw](https://excalidraw.com/) is a great tool for this!
+### Step Two 
+Decide what to assess - defining the scope of the assessment is essential to making it successful. This decision will
+keep you focused and ensure you make forward progress. Helpful questions when thinking about what to assess:
+- What workflows are used most?
+- What workflows does the community want a security assessment of?
+- What expertise does the team doing the assessment have? Does it match any of the potential flows? 
+Try to get matching expertise for the scope!
+### Step Three 
+Now that you have scope defined, we can come up with threats that are reasonable and logical for the projects.
+- Use the [TAG Self-assessment template](https://github.com/cncf/tag-security/blob/main/assessments/guide/self-assessment.md) 
```

Comment:
Are all fields relevant and useful in the TAG-Security Self-Assessment template or would it be useful to create a fork of their template for this project?

Reply:
Fork would be nice especially after we have lessons learnt from vsphere csi-driver assessment too

Reply:
I'll bring this up in the next meeting.

Reply:
I will not do that as part of this PR - but as a follow up. We will fork the TAG-Security one, and then we can iterate on it and make it more suited to our use cases.

Reply:
@cailynse @PushkarJ  just wanted to check in - is there any action I need to take here?

Reply:
For the template I'll open a separate PR once this one is merged.

For this PR we need a /lgtm from you and then an approval from a chair I believe!